scrap_reason_code/migrations/13.0.1.0/pre-migration.py

```diff
@@ -0,0 +1,14 @@
+# Copyright (C) 2019 IBM Corp.
```

Comment:
To make this working for existing installs, please change to '13.0.1.1.0' and change it in manifest too.